OpenRA.Mods.Common/Widgets/ProductionPaletteWidget.cs

```diff
@@ -162,6 +159,9 @@ public override void Initialize(WidgetArgs args)
 
 			hotkeys = Exts.MakeArray(HotkeyCount,
 				i => modData.Hotkeys[HotkeyPrefix + (i + 1).ToString("D2")]);
+
+			overlayFont = Game.Renderer.Fonts[OverlayFont];
+			Game.Renderer.Fonts.TryGetValue(SymbolsFont, out symbolFont);
```

Comment:
Are these changes supposed to be here? The PR description doesn't mention `ProductionPaletteWidget` and the commit message "Apply the configurated values." also doesn't explain it.

Reply:
I suppose so since this moves the assignment to `Initialize` so that it is in line with the change in `SupportPowersWidget`.

Reply:
If you put them in the constructor only the default values are used not the ones configured in .yaml. This was also a surprise to me when I wanted to make use of the new feature.

Reply:
Can we please update the commit message to something more descriptive like "Fix ProductionPalette ignoring yaml font overrides."?

Reply:
I am more of a fan which describes the result/motivation instead of repeating changed file names and literal changes.

Reply:
The issue is that "Apply the configurated values." does not tell us anything, and even after reading the code I wasn't able to guess the result/motivation!

Reply:
Renamed it.